src/bin/devmgr/src/nvme_test.rs

```diff
@@ -57,8 +78,273 @@ impl RequestDriver for NvmeQueue {
     const NUM_IDS: usize = 8;
 }
 
-async fn test<'a>(ctrl: Arc<NvmeController>) {
-    println!("starting req test");
+async fn test3<'a>(ctrl: Arc<NvmeController>) {
```

Comment:
Can you give each test a descriptive name, so we know what it's testing?

Reply:
So, I'm planning to rewrite this whole file into a "real" driver later. This file was used for testing the incremental changes I've been making. I just left it in (not compiled) because I plan to build from it.